indico/modules/rb/client/js/util.jsx

```diff
@@ -238,10 +238,13 @@ export function serializeRecurrenceInfo({type, number, interval}) {
 
 /**
  * Renders the array of recurrence weekdays into a nicely formatted string
+ * @param {object} options Options object
  * @param {array} weekdays Array of weekdays
+ * @param {number} repetition Recurrence repetition (e.g. every `2` weeks)
+ * @param {boolean} [weekdaysOnly=false] Whether to return only the weekdays
```

Comment:
This looks wrong. We don't really build JSDoc, but right now this describes a function with 4 args.

https://jsdoc.app/tags-param - I think "Documenting a destructuring parameter" is what you're looking for:

```
 * @param {object} opts Options
 * @param {array} opts.weekdays Array of weekdays
 * @param {number} opts.repetition Recurrence repetition (e.g. every `2` weeks)
 * @param {boolean} opts.weekdaysOnly Whether to return only the weekdays
```